cmd/mysterium_server/command_run/identity_handler_test.go

```diff
@@ -0,0 +1,48 @@
+package command_run
+
+import (
+	"testing"
+
+	"github.com/mysterium/node/identity"
+	"github.com/mysterium/node/server"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_identityHandler_UseExisting(t *testing.T) {
+	identityManager := identity.NewIdentityManagerFake()
+	client := server.NewClientFake()
+	cache := identity.NewIdentityCacheFake()
+
+	handler := NewNodeIdentityHandler(identityManager, client, cache)
+
+	id, err := handler.UseExisting("address")
```

Comment:
But such value "address" is not mocked in `identityManager`. is not it?

Reply:
This address is ignored in fake identity manager:
```go
func (fakeIdm *idmFake) GetIdentity(_ string) (Identity, error) {
	return fakeIdm.FakeIdentity1, nil
}
```

Reply:
But how can we use *existing* "address" if it dont exist actually. Could could double check, plz

Reply:
Double checked - everything seems ok, not sure what's confusing you.